heuristics/hyades_heuristic.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
I think the first two imports here can be removed as they are not used:

```python
import os 
import numpy
```

Reply:
Agree! They were there in the other heuristic that I got as a template. Im taking them out